.github/ISSUE_TEMPLATE/1.Bug_report.yml

```diff
@@ -0,0 +1,96 @@
+name: Bug Report
+description: Create a bug report for one of the supabase repos or examples.
+labels: ['bug', 'triage']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: A clear description of what the bug is.
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: To Reproduce
+      description: Steps to reproduce the behavior, please provide code snippets or a repository
+      placeholder: |
+        1. Go to '…'
+        2. Click on '…'
+        3. Scroll down to '…'
+        4. See error
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: A clear description of what you expected to happen.
+    validations:
+      required: true
+
+  - type: dropdown
+    id: system-os
+    attributes:
+      label: 'System information: OS'
+      description: Which operating system are you using?
+      options:
+        - Linux
+        - Mac OS
+        - Windows
+    validations:
+      required: true
+
+  - type: dropdown
+    id: system-browser
+    attributes:
+      label: What browser are you seeing the problem on? (if applies)
+      options:
+        - Chrome
+        - Firefox
+        - Safari
+        - Other
+    validations:
+      required: false
+
+  - type: input
+    id: supabase-js-version
+    attributes:
+      label: Supabase JS version
+      description: What version of supabase-js are you using?
+    validations:
+      required: true
+
+  - type: dropdown
```

Comment:
Am I right that the user has to specify a runtime? The issue might not even related to it, but rather the supabase dashboard or some api repository.

Reply:
You're right, i tried to keep the templates that were there, so that only the style changes, and potentially add other information (such as the choice of runtime).

Do you think we should modify these new templates so that they are more representative of Supabase's work? Including, for example, a dropdown to choose what the issue is about (supabase-js, supabase-flutter....)?

Reply:
It depends on where the template gets applied to. In the flutter repos, we currently have our own issue templates. All flutter related issues should be collected there and not in the supabase/supabase repo. So I don't think it makes sense to add a client library dropdown. I think the template here should be very general. Repository specific information should be specified in a template per repository. What do you think?

Reply:
Sounds good, pushing an update asap